extension-config/popup.js

```diff
@@ -0,0 +1,37 @@
+function getActiveTab() {
+    return new Promise((resolve, reject) => {
+      chrome.tabs.query({ active: true, currentWindow: true }, function (tabs) {
+          const activeTab = tabs.find(t => t.active);
+
+          resolve(activeTab);
+      });
+    });
+}
+
+function activateTota11y() {
+    console.log('activating tota11y...');
+    chrome.tabs.executeScript(null, { file: "tota11y.js" });
```

Comment:
Only load the script when the user explicitly clicks the checkbox in the extension popup.